app/Models/InteractionFragment.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class InteractionFragment extends Model {
+
+	protected $table = 'interaction_fragments';
+	public $timestamps = false;
+
+	public function interactionFragment()
+	{
+		return $this->hasMany('App\Models\InteractionFragment', 'enclosing_fragment_id');
+	}
+
+	public function interactionFragment()
```

Comment:
metódy s rovnakým názvom navyše toto večer skúsim implementovať cez Nested Set, takže to možes zmazať ja to doplním